nordic-nrf51822-16k-armcc/scripts/memory_info.py

```diff
@@ -0,0 +1,86 @@
+#! /usr/bin/env python
```

Comment:
I always use this line rather than the more traditional `/usr/bin/python`, and the same for bash. I have always thought about this as more flexible. Have a look [here](http://stackoverflow.com/questions/5709616/whats-the-difference-between-these-two-python-shebangs)